src/mongoose_hooks.erl

```diff
@@ -893,8 +893,10 @@ roster_groups(LServer) ->
       RemoteJID :: jid:jid() | jid:simple_jid(),
       Result :: {mod_roster:subscription_state(), [binary()]}.
 roster_get_jid_info(HostType, ToJID, RemBareJID) ->
-    run_hook_for_host_type(roster_get_jid_info, HostType, {none, []},
-                           [HostType, ToJID, RemBareJID]).
+    Params = #{to_jid => ToJID, remote_jid => RemBareJID},
```

Comment:
I would change names to the following, to keep it consistent with other hooks params:

```suggestion
    Params = #{to => ToJID, remote => RemBareJID},
```